docs/azure/ContinuousIntegration.rst

```diff
@@ -1,198 +1,217 @@
+      export B2C_TENANT="<Primary domain>" # For example, "bifravstciusers"
 
-Link this Azure AD B2C tenant to the subscription for CI by following `the Billing guide <https://docs.microsoft.com/en-us/azure/active-directory-b2c/billing#link-an-azure-ad-b2c-tenant-to-a-subscription>`_.
+#. Link this Azure AD B2C tenant to the subscription for CI by following the `Billing guide <https://docs.microsoft.com/en-us/azure/active-directory-b2c/billing#link-an-azure-ad-b2c-tenant-to-a-subscription>`_.
 
-Create Azure Active Directory B2C application
-*********************************************
+Create the Azure Active Directory B2C application
+*************************************************
 
-Follow the steps in the :ref:`Continous Deployment <azure-continuous-deployment>`  instructions to create a new App registration.
+To create the Azure Active Directory B2C application, complete the following steps:
 
--   Name: Bifravst Web App
--   Redirect URI (make sure to select SPA): ``https://bifravstciapp.z16.web.core.windows.net/`` (instead of ``bifravstciapp`` you need to pick something else that fits your project because this name is globally unique)
+1. Follow the instructions in the :ref:`Continous Deployment <azure-continuous-deployment>` instructions to create a new App registration with the following values:
 
-Note down the *Application (client) ID* and the *Directory (tenant) ID* of the created Active Directory B2C App registration:
+   * Name - Bifravst Web App.
+   * Redirect URI (make sure to select SPA) - ``https://bifravstciapp.z16.web.core.windows.net/`` (Choose a name that fits your project instead of ``bifravstciapp`` since ``bifravstciapp`` is globally unique).
 
-.. code-block:: bash
+#. Export the ``Application (client) ID`` and the ``Directory (tenant) ID`` of the created Active Directory B2C App registration into the ``APP_REG_CLIENT_ID`` and ``B2C_TENANT_ID`` parameters:
 
-    export APP_REG_CLIENT_ID="<application (client) id>"
-    export B2C_TENANT_ID="<Directory (tenant) ID>"
+   .. code-block:: bash
 
-For the test-runner to be able to programmatically log-in users, the resource owner password credentials (ROPC) flow `needs to be enabled <https://docs.microsoft.com/EN-US/azure/active-directory-b2c/configure-ropc?tabs=app-reg-ga>`_ with these settings:
+      export APP_REG_CLIENT_ID="<Application (client) id>"
+      export B2C_TENANT_ID="<Directory (tenant) ID>"
 
--   Name: ``B2C_1_developer``
--   Application claims: select *Show more ...* and then mark *Email Addresses* as a return claim
+#. For enabling the test-runner to programmatically login users, enable `the resource owner password credentials (ROPC) flow <https://docs.microsoft.com/EN-US/azure/active-directory-b2c/configure-ropc?tabs=app-reg-ga>`_ with the following settings:
 
-Add the permission to manager user accounts (Microsoft Graph > ``User.ReadWrite.All``) and grant admin consent.
+   * Name - ``B2C_1_developer``.
```

Comment:
Why are colons not good in these cases?

E.g. later, we use colon do ask for user action as well: 

![image](https://user-images.githubusercontent.com/188915/106743116-3ccc3500-661e-11eb-9d8a-83ae6be1e273.png)


Why is this different?